SI/Sem.02/Solutions/Task03.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	double number1, number2;
+	cin >> number1 >> number2;
+	cin.ignore();
+	double finalResult = number1 > number2 ? number1 : number2;
```

Comment:
If we are receiving 2 different numbers it is correct, however, can we check whether they are equal?